ghu_web/ghu_main/templates/ghu_main/org_profile.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'ghu_main/base.html' %}
+{% load static %}
+{% block title %}Profiles{% endblock %}
+{% block styles %}
+    {{ block.super }}
+    <!-- <link href="{% static "/ghu_main/css/toolkits.css" %}" rel="stylesheet"> -->
+    <link href="{% static "/ghu_main/css/generic-list.css" %}" rel="stylesheet">
+    <style>
+      strong {
+          display:inline !important;
```

Comment:
let's not use `!important` in this project, it's kinda bad practice. if you don't put this, does it not work?

Reply:
ok it doesn't look like it would. you should be able to do something like

    p strong:first-child {
        display: inline;
    }

basically, when two selectors have the same "specificity" (as this does with the corresponding selector in `generic-list.css`, go look at it), the one defined later takes precedence. this one is defined later